docs/core/index.md

```diff
@@ -44,4 +44,4 @@ Hello World!
 
 .NET Core binary distributions are built and tested on Microsoft-maintained servers in Azure and supported just like any Microsoft product.
 
-[Red Hat supports .NET Core](http://redhatloves.net/) on Red Hat Enterprise Linux (RHEL). Red Hat builds .NET Core from source and makes it available in the [Red Hat Software Collections](https://developers.redhat.com/products/softwarecollections/overview/). Red Hat and Microsoft collaborate to ensure that .NET Core works well on RHEL.
+[Red Hat supports .NET Core](https://developers.redhat.com/topics/dotnet/) on Red Hat Enterprise Linux (RHEL). Red Hat builds .NET Core from source and makes it available in the [Red Hat Software Collections](https://developers.redhat.com/products/softwarecollections/overview/). Red Hat and Microsoft collaborate to ensure that .NET Core works well on RHEL.
```

Comment:
ℹ️ this where the link redirects to. The original site does not support https; the redirect does. Will defer to you on whether you want to keep the catch branded URL or the https-compliant one. 👍

Reply:
we usually try to avoid extra redirects but the previous one does look catchy 😄 

@tmds do you know if the previous site ever had any content or was it always a redirect?

Reply:
@mairaw, Its first appearance in the Wayback Machine in Apr 2016 was a redirect to `http://developers.redhat.com/dotnet/?utm_source=event&utm_medium=vanityurl&utm_term=redhatloves.net&utm_campaign=msbuild`

Reply:
yep, I was checking that too. I think in this case the catchy URL could stay since it does redirect to the https one